pkg/storage/ec/client_planet_test.go

```diff
@@ -153,7 +153,10 @@ func newAddressedOrderLimit(action pb.PieceAction, satellite *satellite.Peer, up
 	if err != nil {
 		return nil, err
 	}
-
+	orderExpiration, err := ptypes.TimestampProto(time.Now().Add(24 * time.Hour))
```

Comment:
I suspect we want a larger expiration time than 1day?

Reply:
this is for test only

Reply:
ah, right :)